octoprint_mrbeam/mrb_logger.py

```diff
@@ -158,12 +164,23 @@ def _analytics_log_event(self, level, msg, analytics_id, *args, **kwargs):
         if analytics_handler is not None:
             try:
                 msg = msg % args if args and msg else msg
-                caller = None
-                caller_myself = getframeinfo(stack()[0][0])
+                caller = getframeinfo(stack()[0][0])
+                caller_myself = caller
                 i = 1
```

Comment:
It would be best to remove the use of an index / counter by using a for / if loop:
```
for caller in ( getframeinfo(stack_elm[0]) for stack_elm in stack() ) :
    try:
        if ( _while_condition_ ) :
            # do stuff
    except:
        continue
```